projects/ngrx.io/content/guide/effects/install.md

```diff
@@ -0,0 +1,42 @@
+# Installation
+
+## Installing with `npm`
+
+For more information on using `npm` check out the docs <a href="https://docs.npmjs.com/cli/install" target="_blank">here</a>.
+
+```sh
+npm install @ngrx/effects --save
+```
+
+## Installing with `yarn`
+
+For more information on using `yarn` check out the docs <a href="https://yarnpkg.com/docs/usage" target="_blank">here</a>.
+
+```sh
+yarn add @ngrx/effects
+```
+
+## Installing with `ng add`
+
+If your project is using the Angular CLI 6+ then you can install the Effects to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
+
+```sh
+ng add @ngrx/effects
+```
+### Optional `ng add` flags
+
+* path - path to the module that you wish to add the import for the `EffectsModule` to.
+* flat - Indicate if a directoy is to be created to hold your effects file
+* spec - Specifies if a spec file is generated.
+* project - name of the project defined in your `angular.json` to help locating the module to add the `EffectsModule` to.
+* module - name of file containing the module that you wish to add the import for the `EffectsModule` to. Can also include the relative path to the file. For example, `src/app/app.module.ts`;
```

Comment:
```suggestion
* module - name of file containing the module that you wish to add the import for the `EffectsModule` to. Can also include the relative path to the file. For example, `src/app/app.module.ts`
```

Reply:
This should be corrected in latest commit